### PR TITLE
Add to DropItem(Item item, bool isThrown)

### PR DESCRIPTION
**Pull request by Rysik5318**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1864,6 +1864,11 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
             if (broadcast.Show)
                 Broadcast(broadcast.Duration, broadcast.Content, broadcast.Type, shouldClearPrevious);
         }
+        /// <summary>
+        /// Drops an item from the player's inventory.
+        /// </summary>
+        /// <param name="item">The item to be dropped.</param>
+        public void DropItem(Item item, bool isThrown = false) => Inventory.UserCode_CmdDropItem__UInt16__Boolean(item.Serial, isThrown);
```
Comment (VALERA771) [inline Exiled.API/Features/Player.cs:1871]: Will be ambiguous reference

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -1872,6 +1877,16 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
         /// <returns>dropped <see cref="Pickup"/>.</returns>
         public Pickup DropItem(Item item) => Pickup.Get(Inventory.ServerDropItem(item.Serial));
 
+        /// <summary>
+        /// Drops the held item. Will not do anything if the player is not holding an item.
+        /// </summary>
+        /// <seealso cref="CurrentItem"/>
+        public void DropHeldItem(bool isThrown = false)
```
Comment (VALERA771) [inline Exiled.API/Features/Player.cs:1884]: Same as above

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (Rysik5318) [reply Exiled.API/Features/Player.cs:1871]: So what?

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (Rysik5318) [reply Exiled.API/Features/Player.cs:1884]: So what??

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (VALERA771) [reply Exiled.API/Features/Player.cs:1871]: Plugin will require recompile => breaking change. Default parameter's value should be removed

File: Exiled.API/Features/Player.cs (hunk 2 above)
Comment (louis1706) [reply Exiled.API/Features/Player.cs:1884]: remove = false

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -1872,6 +1877,16 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
         /// <returns>dropped <see cref="Pickup"/>.</returns>
         public Pickup DropItem(Item item) => Pickup.Get(Inventory.ServerDropItem(item.Serial));
```
Comment (louis1706) [inline Exiled.API/Features/Player.cs:1878]:
```suggestion
        public Pickup DropItem(Item item) => item is not null ? Pickup.Get(Inventory.ServerDropItem(item.Serial)) : null;
```
I know this is not directly related to your PR but would be nice

File: Exiled.API/Features/Player.cs (hunk 1 above)
Comment (louis1706) [inline Exiled.API/Features/Player.cs:1871]:
```suggestion
        public void DropItem(Item item, bool isThrown) => item is not null ? Inventory.UserCode_CmdDropItem__UInt16__Boolean(item.Serial, isThrown) : null;
```

File: Exiled.API/Features/Player.cs [hunk 4]
```diff
@@ -1872,6 +1877,16 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
...
 
+        /// <summary>
+        /// Drops the held item. Will not do anything if the player is not holding an item.
+        /// </summary>
+        /// <seealso cref="CurrentItem"/>
+        public void DropHeldItem(bool isThrown = false)
+        {
+            if (CurrentItem is null) return;
+
```
Comment (louis1706) [inline Exiled.API/Features/Player.cs:1887]:
```suggestion
```

**Review by louis1706 — APPROVED** [review]

**Review by louis1706 — CHANGES_REQUESTED** [review]
Fix BuildError

**Comment by Rysik5318** [comment]
> Fix BuildError
God, I'm so sick of this BuildError.

**Review by louis1706 — APPROVED** [review]

**Pull request merged**